lib/node_modules/@stdlib/math/base/special/cscdf/lib/main.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var f32 = require( '@stdlib/number/float64/base/to-float32' );
+var sindf = require( '@stdlib/math/base/special/sindf' );
+
+
+// MAIN //
+
+/**
+* Computes the cosecant of a single-precision floating-point number (in degrees).
+*
+* @param {number} x - input value (in degrees)
+* @returns {number} cosecant
+*
+* @example
+* var v = cscdf( 30.0 );
+* // returns ~2.0
+*
+* @example
+* var v = cscdf( 45.0 );
+* // returns ~1.41
+*
+* @example
+* var v = cscdf( 60.0 );
+* // returns ~1.15
+*
+* @example
+* var v = cscdf( 90.0 );
+* // returns 1.0
+*
+* @example
+* var v = cscdf( 0.0 );
+* // returns Infinity
+*
+* @example
+* var v = cscdf( NaN );
+* // returns NaN
+*/
+function cscdf( x ) {
+	return f32( 1.0 ) / sindf( f32( x ) );
```

Comment:
The entire division operation should be wrapped in f32() to avoid returning float64 value:

```suggestion
	return f32( f32( 1.0 ) / sindf( f32( x ) ) );
```

Reply:
Yes! Thanks @Planeshifter for the review!